src/common/com/intellij/plugins/haxe/ide/info/HaxeParameterInfoHandler.java

```diff
@@ -81,14 +81,10 @@ public void showParameterInfo(@NotNull PsiElement element, CreateParameterInfoCo
   @Nullable
   private static HaxeFunctionDescription tryGetDescription(HaxeCallExpression callExpression) {
     final PsiElement target = ((HaxeReference)callExpression.getExpression()).resolve();
-    final PsiElement targetParent = target == null ? null : target.getParent();
-    if (target instanceof HaxeComponentName && targetParent instanceof HaxeNamedComponent) {
-      final HaxeReference[] references = PsiTreeUtil.getChildrenOfType(callExpression.getExpression(), HaxeReference.class);
-      final HaxeClassResolveResult resolveResult = (references != null && references.length == 2)
-                                                   ? references[0].resolveHaxeClass()
-                                                   : HaxeClassResolveResult
-                                                     .create(PsiTreeUtil.getParentOfType(callExpression, HaxeClass.class));
-      return HaxeFunctionDescription.createDescription((HaxeNamedComponent)targetParent, resolveResult);
+    if (target instanceof HaxeNamedComponent) {
+      final HaxeClass targetParent = (HaxeClass) ((HaxeMethod) target).getContainingClass();
```

Comment:
targetParent should be target.getContainingClass() ... not target.getParent().
Earlier (before our PSI fixes), target.getParent() used to return HaxeClass.
it now returns HaxeClassBody... hence, it should be getContainingClass()